reconcile/utils/terrascript_aws_client.py

```diff
@@ -5213,10 +5278,19 @@ def populate_tf_resource_s3_cloudfront_public_key(self, spec):
         self.add_resources(account, tf_resources)
 
     def _get_alb_target_ips_by_openshift_service(
-        self, identifier, openshift_service, account_name, namespace_info, ocm_map
-    ):
+        self,
+        identifier: str,
+        openshift_service: str,
+        account_name: str,
+        namespace_info: Mapping[str, Any],
+        ocm_map: OCMMap | None,
+    ) -> set[str]:
         account = self.accounts[account_name]
```

Comment:
why not mark ocm_map as not None in param?

Reply:
can be none, I need to check somewhere if this is none. see [this](https://github.com/jfreixa/qontract-reconcile/blob/e8a67af2849e94f2d2354cb855c3f930969dec19/reconcile/terraform_resources.py#L267C14-L267C15)

Reply:
https://github.com/app-sre/qontract-reconcile/blob/b8eb92e4fec51d7e71badf7ca38a0a3dfa523daf/reconcile/utils/terrascript_aws_client.py#L5423-L5426 this is the only usage, move None check there to make this function type clean.